10-JavaScript/welcome-to-js.md

```diff
@@ -243,6 +243,29 @@ To point out the three components:
 
 **Question:** What Ruby code could you use to achieve the same thing?
 
+#### more `for` Loops
+There are a few other types of for loops that are useful. These are a bit more similar to the `each` loops we are used to seeing in Ruby.
+
+**`for`...`in`...**
+
+This `for` loop gives us access to each element's key within a collection. With an array, this is each element's index. This isn't so useful.
```

Comment:
Since the `for...of` loop is more immediately useful / familiar from the Ruby curriculum, it might make sense to start with that.

I also think this section on for/in could use a little re-wording, to approach it from "where is it useful" as opposed to "where is it not useful". Maybe something like:

The `for...in` loop iterates over the keys of a collection. This is most useful when working with an object (similar to a Ruby hash). We haven't discussed objects in detail yet, but the following syntax should be familiar:

```js
let pets = {
  kari: "Kylo",
  dan: "Gecky",
  jamie: "Octo and Amper"
};
for (let human in pets) {
  console.log(`${ human }'s pets: ${ pets[human] }`);
}
// kari
// dan
// jamie
```

**Question:** What do you get when you use a `for...in` loop on an array? Why might this be useful?

Reply:
Ok - I was trying not to make this any longer than it needs to be, but I think this example is great!